src/ashley/templates/forum/forum_list.html

```diff
@@ -77,6 +77,7 @@ <h3 class="m-0 card-title h5 text-dark">{% trans "Forums" %}</h3>
         {% blocktrans trimmed with poster_url=poster_url username=node.last_post.poster|forum_member_display_name %}
         By: <a href="{{ poster_url }}">{{ username }}</a>
         {% endblocktrans %}
+        {% include "partials/user_icon.html" with topic=node.last_post.topic user=node.last_post.poster %}
```

Comment:
I would have added a minimal integration test that renders the forum list and confirms the presence of the icon 
Same for other places where you add the icon :man_shrugging:

Reply:
ok :-) I had them on the first version because of the way I had build it and was frustrated that it wasn't checked anymore that was probably the reason why, I still had to do them :-)

Reply:
Sam I've added tests for this purpose, I've resubmitted a new review for this. thanks in advance